certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/Main.kt

```diff
@@ -134,9 +134,10 @@ class Main : Callable<Int> {
   private fun parsePemCertificate(file: File): Certificate {
     try {
       val data = file.readText()
-          .replace("-----BEGIN CERTIFICATE-----\n", "")
-          .replace("-----END CERTIFICATE-----\n", "")
+          .replace("-----BEGIN CERTIFICATE-----\r?\n".toRegex(), "")
+          .replace("-----END CERTIFICATE-----\r?\n.*".toRegex(), "")
```

Comment:
what»s the `.*` for?

Reply:
extra new lines, although I seem to remember one of your files have CERTIFICATE followed by PRIVATE KEY?

Reply:
Going to change this to extract the middle part instead of replacing.